### PR TITLE
Added Swagger support

### PR DESCRIPTION
**Pull request by alecstincu**
### What does it fix?

https://github.com/code4romania/next-door/issues/29

### How has it been tested?

Manually by accessing /swagger-ui.html and making sure information is in place.

### CODE REVIEW

File: api/build.gradle [hunk 1]
```diff
@@ -19,6 +19,9 @@ dependencies {
     implementation 'org.springframework.boot:spring-boot-starter-security'
     implementation 'io.jsonwebtoken:jjwt:0.9.1'
 
+    compile group: 'io.springfox', name: 'springfox-swagger2', version: '2.9.2'
```
Comment (georgebejan) [inline api/build.gradle]: `compile` was deprecated, use `implementation` instead

**Review by georgebejan — CHANGES_REQUESTED** [review]

File: api/build.gradle (hunk 1 above)
Comment (alecstincu) [reply api/build.gradle]: done

**Comment by georgebejan** [comment]
@alecstincu something went wrong with your rebase since now you have unnecessary files changed. Please rebase again

**Comment by alecstincu** [comment]
will create a new one

**Pull request closed without merging**